crates/turborepo-lib/src/commands/mod.rs

```diff
@@ -102,4 +104,14 @@ impl CommandBase {
             Ok(None)
         }
     }
+
+    pub fn daemon_file_root(&self) -> PathBuf {
+        std::env::temp_dir().join("turbod").join(self.repo_hash())
+    }
+
+    fn repo_hash(&self) -> String {
+        let mut hasher = Sha256::new();
+        hasher.update(self.repo_root.to_str().unwrap().as_bytes());
+        hex::encode(&hasher.finalize()[..8])
```

Comment:
The Go implementation is 16 characters, I didn't check in to see what this did but the eight stands out to me. Can we get a very simple test of this function to demonstrate the behavior?

(I'm also hoping that `std::env::temp_dir()` and Go's `os.TempDir()` are 1:1, but if they're not, oh well.)

***

Also, in a future world, I'd prefer if this had an identifier that was distinct from file path. Something like:

`git rev-list --max-parents=0 --topo-order HEAD`

The first SHA spit out by this would be perfect for addressing.

(Failing `git` would fallback to file path.)

Reply:
If we use something other than file path, we need to account for multiple copies of the same repository.